grunttasks/jscs.js

```diff
@@ -6,13 +6,23 @@ module.exports = function (grunt) {
   'use strict';
 
   grunt.config('jscs', {
-    src: [
-      '{,<%= yeoman.app %>/**/,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
-      '!<%= yeoman.app %>/bower_components/**',
-      '!<%= yeoman.app %>/scripts/vendor/**'
-    ],
-    options: {
-      config: '.jscsrc'
+    server: {
+      src: '{,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
+      options: {
+        config: '.jscsrc'
+      }
+    },
+    app: {
+      src: [
+        '{,<%= yeoman.app %>/**/,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
```

Comment:
I think you are right, I just added the server src, so it might pretty much be redundant, and the commit message was keeping in mind the jscs requirePaddingNewLinesAfterUseStrict: true rule, i will change that :)